src/App.jsx

```diff
@@ -14,57 +15,78 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
+  const handleSelectedGood = good => {
+    if (selectedGood === good) {
+      setSelectedGood('');
+    } else {
+      setSelectedGood(good);
+    }
+  };
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  const reset = () => setSelectedGood('');
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button data-cy="AddButton" type="button" className="button">
-              +
-            </button>
-          </td>
+  return (
+    <main className="section container">
+      {selectedGood === '' ? (
+        <h1 className="title is-flex is-align-items-center">
+          No goods selected
+        </h1>
+      ) : (
+        <h1 className="title is-flex is-align-items-center">
+          {selectedGood} is selected
+          <button
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+            onClick={reset}
+          />
+        </h1>
+      )}
+      <table className="table">
+        <tbody>
+          {goods.map(good => {
+            const buttonClickHandler = () => handleSelectedGood(good);
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+            return (
+              <tr
+                data-cy="Good"
+                key={good}
+                className={
+                  good === selectedGood ? 'has-background-success-light' : ''
+                }
```

Comment:
Use the classnames library for add classes with condition
```suggestion
                className={
                  isSelected ? 'has-background-success-light' : ''
                }
```